data/json/items/tool/metalworking.json

```
@@ -221,6 +221,29 @@
     ],
     "magazine_well": "1000 ml"
   },
+  {
+    "id": "hydraulic_press",
+    "type": "TOOL",
+    "category": "tools_workshop",
+    "name": { "str": "hydraulic press" },
+    "description": "A large, barely portable hydraulic press, that uses a 36kW motor to squash whatever needs squashing. Do not put your hand in here, it will get squashed.",
```

```suggestion
    "description": "A large, barely portable hydraulic press, that uses a 36kW motor to squash whatever needs squashing.  Do not put your hand in here, it will get squashed.",
```
For one reason or another, BN mandates double-space after periods and the like in sentences.